lib/lita/handlers/reviewme.rb

```diff
@@ -96,16 +96,16 @@ def comment_on_github(response)
         end
       end
 
-      def mention_reviewer(response)
+      def mention_reviewer(response, room: get_room(response))
         url = response.matches.flatten.first
-        reviewer = next_reviewer
+        reviewer = next_reviewer(room)
         response.reply(chat_mention(reviewer, url))
       end
 
       private
 
-      def next_reviewer
-        redis.rpoplpush(REDIS_LIST, REDIS_LIST)
+      def next_reviewer(room)
+        ns_redis(room).rpoplpush(REDIS_LIST, REDIS_LIST)
```

Comment:
I think that's actually something silly like `room_object` https://github.com/litaio/lita/blob/94ad234c9737d658d1b0d8d7829530e4b378dc62/lib/lita/message.rb#L30